src/libraries/System.Private.CoreLib/src/System/Buffers/Text/Base64Validator.cs

```diff
@@ -0,0 +1,145 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Buffers.Text
+{
+    public static partial class Base64
+    {
+        /// <summary>
+        /// Validates the span of UTF-8 encoded text represented as base64 into binary data.
+        /// </summary>
+        /// <param name="base64Text">The input span which contains UTF-8 encoded text in base64 that needs to be validated.</param>
+        /// <returns>true if <paramref name="base64Text"/> is decodable; otherwise, false.</returns>
+        public static bool IsValid(ReadOnlySpan<char> base64Text) =>
+            IsValid<char, Base64CharValidatable>(base64Text, out _);
+
+        /// <summary>
+        /// Validates the span of UTF-8 encoded text represented as base64 into binary data.
```

Comment:
The `char` methods probably shouldn't talk about "UTF-8 encoded" inputs

Reply:
Fixed